Sources/swiftpm-xctest-helper/main.swift

```diff
@@ -0,0 +1,108 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+#if os(OSX)
+
+import XCTest
+import Foundation
+import Darwin.C
+
+enum Error: ErrorProtocol {
+    case invalidUsage
+    case unableToLoadBundle(String)
+    case couldNotOpenOutputFile(String)
+    case couldNotConvertStringToNSData(String)
+}
+
+extension String {
+    func normalizedPath() -> String {
+        var path = self
+        if !(path as NSString).isAbsolutePath {
+            path = NSFileManager.default().currentDirectoryPath + "/" + path
+        }
+        return (path as NSString).standardizingPath
+    }
+}
+
+do {
+    guard Process.arguments.count == 3 else {
+        throw Error.invalidUsage
+    }
+    let bundlePath = Process.arguments[1].normalizedPath()
+    let outputFile = Process.arguments[2].normalizedPath()
+
+    // Bundle may write to stdout when it is being loaded. 
+    // Client should decide what to do with the output.
```

Comment:
These aren't complete sentences which makes the intent a little unclear. Something like:
```
// Note that the bundle might write to stdout while it is being loaded, but we don't try to handle that here. Instead the client should decide what to do with any extra output from this tool.
```
would solve both.